patches/ember-cli-addon-docs@6.0.2.patch

```diff
@@ -1,7 +1,16 @@
 diff --git a/index.js b/index.js
-index f91f1ea4d0c64c41595f7b27dca4d6618145cd0d..be9a6b5df08e0635b73e70d497ae82503f014a68 100644
+index f91f1ea4d0c64c41595f7b27dca4d6618145cd0d..e788b5fdb14f723441c08db1921626c2f021026e 100644
 --- a/index.js
 +++ b/index.js
+@@ -129,7 +129,7 @@ module.exports = {
+     if (!includer.options.snippetSearchPaths) {
+       if (this._documentingAddonAt()) {
+         // we are a standalone app, so our code is here
+-        includer.options.snippetSearchPaths = ['app'];
++        includer.options.snippetSearchPaths = ['app',`${this._documentingAddonAt()}/dist`];
```

Comment:
Another fix on ember-cli-addon-docs. If the snippet is in the addon source files, it does not work. I patched-it manually. But maybe we could just move the snippet out of the addon at the cost of some maintenance when the snippet change.